### DISCUSSION TITLE
VS CODE DOESN'T RUN ON LOCALHOST BROWSER

### DISCUSSION THREAD

**Discussion by samora254** [coder/code-server#7054] [General]
this is probably a very amateur question (because I am totally new to coding). I am using pythagora to write code, but when I say 'Go Live' to preview the work, the localhost only shows a file/folder structure, not the app being developed. Need help to configure it correctly. I'm using PC. thanks. 
![image](https://github.com/user-attachments/assets/d4fa2f9f-0e46-4a50-b819-6ae700cc6586)

**Comment by code-asher** [comment]
Since you are directly accessing your app on port 5500, it does not seem like code-server is involved.  I can only guess that your application is serving files instead of whatever it is supposed to be doing.  I am not familiar with Pythagora unfortunately, you might get better answers from them!
